### ISSUE TITLE
Support Zotero group libraries in source editor

### ISSUE THREAD

**Issue by jooyoungseo** [quarto-dev/quarto#143]
After `Quarto: Zotero - Connect Web Library`, users can insert citations (Ctrl+Shift+F8) in Visual Editor.

From accessibility standpoint, source editor is more accessible to screen readers. However, `@` sign only triggers autocompletion for a local bib file content. It would be nicer if the connected Zotero Group Libraries can be popped up in this autocompletion.

**Comment by jjallaire** [comment]
Agreed! Unfortunately the Zotero features are fairly coupled to the visual editor right now. We will remedy this later on this year.

**Comment by chillinflute** [comment]
I think this suggestion is still valuable. In my current workflow at least I have to keep swapping between source and visual editor to add Zotero citations because of this imparity.
